index.js

```diff
@@ -1,17 +1,66 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.length = this.items.length;
+    function compare(a, b) {
+      return a - b;
+    }
+    this.items.sort(compare);
+    // Este tambien nos valdría
+    // this.min(this.items);
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (this.items[pos]) {
+      return this.items[pos];
+    } else {
+      throw new Error("OutOfBounds");
+    }
+  }
 
-  max() {}
+  max() {
+    function compare(a, b) {
+      return b - a;
+    }
+    if (this.items.length > 0) {
+      this.items.sort(compare);
+      return this.items[0];
+    } else {
+      throw new Error("EmptySortedList");
+    }
```

Comment:
Todo esto no te haria falta, ya que en la propia funcion de add estas haciendo el sort, por ello lo unico que tienes que hacer es saber como lo estas ordenando, en este caso esta ordenando de manera ascendente, por ello, cogiendo la ultima posicion lo tendrias
```suggestion
    if (this.items.length) {
      return this.items[this.length - 1];
    } else {
      throw new Error("EmptySortedList");
    }
```